packages/context/src/__tests__/ContextSystemProvider.test.js

```diff
@@ -44,7 +44,7 @@ describe('props', () => {
 
 		const contextValue = {
 			Olaf: {
-				css: [{ background: 'white' }, `padding: 20px`],
+				css: [{ background: 'white' }, `padding: 20px;`],
```

Comment:
This is something to be aware of. Originally this test broke on a syntax error from PostCSS I believe. One small difference here between `cssjanus` and `rtlcss`.

Reply:
I poked around, checking the compiled CSS for various components. I don't the think ending `;` affects anything.
The only things this *may* affect would be other stylis plugins (like CSS variables or specificity).

It doesn't appear to be the case 👍